src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -70,11 +82,21 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, location
     # if the user had not logged in.
     check_kube_connection(configuration)
 
```

Comment:
What if there is an error here?

Reply:
We have try catch and telemetry logging inside the function